components/proxy/src/main/java/com/hotels/styx/ServerConfigSchema.java

```diff
@@ -73,12 +73,17 @@
                                     optional("bossThreadsCount", integer()),
                                     optional("clientWorkerThreadsCount", integer()),
                                     optional("workerThreadsCount", integer()),
+                                    // tcpNoDelay is deprecated by PR #464
                                     optional("tcpNoDelay", bool()),
+                                    // nioReuseAddress is deprecated by PR #464
                                     optional("nioReuseAddress", bool()),
+                                    // nioKeepAlive is deprecated by PR #464
                                     optional("nioKeepAlive", bool()),
+                                    // maxInitialLength is deprecated by PR #464
```

Comment:
`maxInitialLength` is still used. Please remove the comment.

Reply:
Done